<launch>
    <node name="map_traverser" pkg="wifi_localization" type="map_traverser" output="screen">
        <!--
        <rosparam param = "x_positions">[-1.64, -0.0594, 1.37, 2.57, 4.07, 5.49, 6.84, 8.25, 9.74, 11.7, 13.7, 15.2, 17.4, 19.9, 22.1, 23.1, 24.9, 27.2, 29.6, 30.9, 32.8, 34.3, 35.8, 37.8, 39.00, 41.00, 42.8, 44.00, 45.1]</rosparam>
        <rosparam param = "y_positions">[-3.37, -3.01, -2.7, -2.25, -1.83, -1.4, -1.02, -0.695, -0.262, 0.408, 0.914, 1.48, 2.28, 3.19, 4.01, 4.52, 5.26, 6.00, 6.77, 7.64, 8.36, 9.08, 9.75, 10.7, 11.3, 12.3, 13.1, 13.8, 14.3]</rosparam>
        -->
        <rosparam param = "x_positions">[0.972, -0.0697, -0.996, -1.63, -2.66, -3.68, -4.55, -5.37, -6.31, -7.21, -8.46, -8.89, -9.95, -10.9, -11.7, -12.9, -14, -15, -16.2, -16.8]</rosparam>
        <rosparam param = "y_positions">[9.94, 7.2, 4.75, 3.33, 0.593, -1.86, -4.1, -6.53, -9.36, -12.1, -14.2, -16.8, -19.3, -22.0, -23.8, -26.8, -29.2, -32.1, -34.4, -36.6]</rosparam>
        <rosparam param = "angle_z">1.0</rosparam>
        <rosparam param = "angle_w">0.0</rosparam>
    </node>
</launch>